packages/grafana-ui/src/components/ButtonCascader/ButtonCascader.story.tsx

```diff
@@ -24,24 +28,24 @@ const getKnobs = () => {
         ],
       },
       { label: 'D', value: 'D' },
-    ]),
-  };
+    ],
+  },
+  argTypes: {
+    icon: { control: { type: 'select', options: ['plus', 'minus', 'table'] } },
+    options: { control: 'object' },
+    className: NOOP_CONTROL,
+    value: NOOP_CONTROL,
+    fieldNames: NOOP_CONTROL,
+  },
 };
 
-export const simple = () => {
-  const { disabled, text, options } = getKnobs();
-  return (
-    <ButtonCascader disabled={disabled} options={options} value={['A']}>
-      {text}
-    </ButtonCascader>
-  );
+const Template: Story<ButtonCascaderProps> = (args) => {
+  return <ButtonCascader {...args}>{args.children}</ButtonCascader>;
```

Comment:
nit: we probably shouldn't spread `args.children` onto the ButtonCascader. We can separate the two by doing something like:

```
const Template: Story<ButtonCascaderProps> = ({ children, ...args }) => {
  return <ButtonCascader {...args}>{children}</ButtonCascader>;
};
```